src/dualize.jl

```diff
@@ -50,4 +46,35 @@ function dualize(primal_model::MOI.ModelLike, dual_problem::DualProblem{T}, dual
                                   primal_objective, con_types)
 
     return dual_problem
-end
\ No newline at end of file
+end
+
+# JuMP dualize
+function dualize(model::JuMP.Model; dual_names::DualNames = DualNames("", ""))
+    # Create an empty JuMP model
+    JuMP_model = JuMP.Model()
+
+    if JuMP.mode(model) != JuMP.AUTOMATIC # Only works in AUTOMATIC mode
+        error("Dualization does not support solvers in $(model.moi_backend.mode) mode")
+    end
+    # Dualize and attach to the model
+    dual_problem = dualize(backend(model); dual_names = dual_names)
+    MOI.copy_to(backend(JuMP_model), dual_problem.dual_model)
```

Comment:
Why not `dualize(backend(model), DualProblem(backend(JuMP_model), PrimalDualMap{T}()); dual_names = dual_names)` to save a copy ?

Reply:
No reason, we can do it like this, but I would merge the other PR first because it may interfere in this one.